Automattic/jetpack
# Sync my jetpack's Boost page with Boost's getting started page

Fixes https://github.com/Automattic/jetpack/issues/38947.

After some discussions, we decided to use the data My Jetpack uses to render its Boost pricing page ([class-boost.php](https://github.com/Automattic/jetpack/blob/update/my-jetpack-boost-getting-started-pages/projects/packages/my-jetpack/src/products/class-boost.php)), as the react component in My Jetpack isn't reusable.

## Proposed changes:

* Sync features list between My Jetpack's Boost page and Boost's getting-started page;
* Add data sync option to get data about Boost's features list/tiers from My Jetpack;
* Update Boost's getting-started page to rely on new data sync option;
* Add test to make sure no accidental updates to Boost's features list/tiers don't happen in My Jetpack.

### Other information:


## Jetpack product discussion



n/a

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Check Boost's getting started page to make sure it properly lists all features - http://jetpack-boost.test/wp-admin/admin.php?page=jetpack-boost#getting-started;
* Check my jetpack's add/boost page to make sure feature list is the same as Boost's - http://jetpack-boost.test/wp-admin/admin.php?page=my-jetpack#/add-boost (might not work if you're offline).
